source/extensions/quic/connection_id_generator/envoy_deterministic_connection_id_generator.h

```diff
@@ -1,7 +1,13 @@
 #pragma once
 
+#include "source/common/quic/envoy_quic_connection_id_generator_factory.h"
+
 #include "quiche/quic/core/deterministic_connection_id_generator.h"
 
+#if defined(__linux__)
+#include <linux/filter.h>
+#endif
+
 namespace Envoy {
 namespace Quic {
 
```

Comment:
Please note that there is a new virtual function in ConnectionIdGeneratorInterface called ConnectionIdLength(uint8_t first_byte). I'm not sure when it will make its way into Envoy Quiche.

That said, there's no need to override the implementation in DeterministicConnectionIdGenerator, so I don't think there's any code change necessary in this PR.